backend/src/server_host.py

```diff
@@ -313,6 +314,11 @@ async def shutdown(request: Request):
     return json(success_response())
 
 
+@app.get("/status")
+async def status(request: Request):
+    return json({"ready": backend_ready})
```

Comment:
Quick question: what does `ready: true` mean? As is: what state of the server does this represent?

Reply:
That means that the backend is done with all its initial setup, ie installing dependencies and starting the worker